lib/text.rb

```diff
@@ -0,0 +1,19 @@
+require 'twilio-ruby'
+
+class Text
+
+  def send_sms
+    account_sid = 'ACb72247efd4758c41dbdbdedf140fc3e7'
```

Comment:
This information (sid, token and phone numbers) should be hidden by using an environment variable. At the moment, someone could access your code and hack into your account (i think) but an env variable means you can gitignore the file and it wont show on github.